packages/static-hosting/lib/static-hosting.ts

```diff
@@ -53,6 +54,9 @@ export class StaticHosting extends Construct {
 
     constructor(scope: Construct, id: string, props: StaticHostingProps) {
         super(scope, id);
+        
+        // Should the stackExportPrefix is empty, 'StaticHosting' should be used as the prefix 
+        const exportPrefix =  props.exportPrefix ? props.exportPrefix :  'StaticHosting'
```

Comment:
What's the advantage of having this default to `StaticHosting` rather than just a blank string?

Reply:
Well this is act as an identifier, say if it is blank string: The bucket name would be exported as `BucketName`, which perhaps you may have already using `BucketName` as an export in one of your other stack.

Reply:
Oh okay that makes sense :slightly_smiling_face: